### PR TITLE
Yc 551 b - filter by subdomains

### PR DESCRIPTION
**Pull request by monodo**
rebase @felicemcc branch yc-551 as main branch has changed a lot

### CODE REVIEW

**Review by danickfort — APPROVED** [review]
Looks good to me 👍 
⚠️ Make sure to add the Site->Administrative Entity many-to-many entries in the DB table after deploying

File: permits/tests/test_permit_request_api.py [hunk 1]
```diff
@@ -381,28 +381,6 @@ def test_qgisserver_is_up_and_atlas_plugin_is_working(self):
...
-            "EXP_FILTER": "permit_request_id in(1)",
-            "PERMIT_REQUEST_ID": 1,
-        }
-
-        qgisserver_url = "http://qgisserver/ogc/?" + urllib.parse.urlencode(values)
-        qgisserver_response = requests.get(
-            qgisserver_url, headers={"Accept": "application/pdf"}, stream=True
-        )
-        self.assertEqual(qgisserver_response.status_code, 200)
-
```
Comment (monodo) [inline permits/tests/test_permit_request_api.py:405]: OK as the whole print process will change in a short time

File: permits/views.py [hunk 2]
```diff
@@ -806,55 +807,66 @@ def anonymous_permit_request(request):
...
-            request.session["entityfilter"] if "entityfilter" in request.session else []
+    # Manage redirect to type step when only 1 item is shown
+    # Handle single tag filters combinations
+    entityfilter = (
+        request.session["entityfilter"] if "entityfilter" in request.session else []
+    )
+
+    entities = services.get_administrative_entities(request.user, current_site)
+
+    # Manage enteties by tag and by site. If site has already 1 result, dont check the tag
```
Comment (monodo) [inline permits/views.py:825]: entities

**Review by monodo — COMMENTED** [review]
rebase + pre-commit

**Pull request merged**